server/src/concepts/user/model/UserValidation.js

```diff
@@ -0,0 +1,30 @@
+const Joi = require("joi-oid");
+Joi.objectId = require("joi-objectid")(Joi);
+
+const validateUser = (user) => {
+  const schema = Joi.object({
+    id: Joi.objectId().required(),
+
+    email: Joi.string()
+      .min(6)
+      .max(255)
+      .required()
+      .email({ minDomainSegments: 2, tlds: { allow: ["com", "net"] } }),
+
+    password: Joi.string().min(6).max(255).required(),
+
+    isAdmin: Joi.boolean().required(),
+
+    phoneNumber: Joi.number().optional(),
+
+    city: Joi.string().optional(),
+
+    isActive: Joi.boolean().required(),
+
+    favourites: Joi.array().default([]).items(Joi.string()),
```

Comment:
To rethink, this field also will be by default empty array when user would be creating, so I think we do not need that here.